README.md

```diff
@@ -146,6 +147,62 @@ Braintree has some [excellent documentation](https://developers.braintreepayment
 
 For additional information check out [Apple's documentation](https://developer.apple.com/reference/applepayjs/) and [Braintree's documentation](https://developers.braintreepayments.com/guides/apple-pay/client-side/javascript/v3).
 
+## Venmo
+There are two ways to utilize Venmo within this extension:
+- Braintree's native Venmo integration
+  - Offers vaulting of payment sources
+- As a finance option for PayPal
+  - Can only be used checkout flow
```

Comment:
This point is not super clear to me, maybe it's just missing some words or it's me missing something?